src/bodywork/workflow_execution.py

```diff
@@ -183,6 +177,31 @@ def run_workflow(
             rmtree(cloned_repo_dir, onerror=_remove_readonly)
 
 
+def _setup_namespace(config) -> str:
+    """Creates namespace to run workflow in.
+
+    :param config: Bodywork config.
+    :return: Name of namespace.
+    """
+    namespace = ""
```

Comment:
`namespace = config.project.namespace is config.project.namespace else config.project.name`

Reply:
Is MyPy being a PITA again?

Reply:
Yup, pure MyPy PITA

Reply:
I think the suggestion above should solve it, maybe `namespace: str = ...` is required.

Reply:
It's not required but either way this is a more elegant solution using the inline if statement. I use them in C# so I don't know why I neglect them in Python 🤷